smac/optimizer/smbo.py

```diff
@@ -157,26 +159,7 @@ def start(self):
         if self.stats.ta_runs == 0 and self.incumbent is None:
             self.logger.info('Running initial design')
             # Intensifier initialization
-            self.incumbent = self.initial_design.run()
-
-        elif self.stats.ta_runs > 0 and self.incumbent is None:
```

Comment:
These were checks to ensure that there is an incumbent selected in the initial design, before the main BO loop. Now initial design only returns configurations, so these checks were unnecessary. It did not seem relevant to push these somewhere inside the BO loop either. So I removed them.

Reply:
No, these checks run instead of the initial design (please re-check the if/elif/elif/else clause). So I think they are quite relevant here and should be kept at the exact same location.